Data/Help/1/script.html

```diff
@@ -361,7 +361,7 @@ <h3><u>Character Script Commands</u></h3>
       selections.</li>
   <li><a name="waitforitem"><b>Wait for item</b>
     </a> - Wait until the specified element is located in the marked room region.</li>
-  <li><a name="logical-wait"><b>Logical Waits</b></a> - Logical wait commands can be used to combine multiple queries into a single query. A logical wait block begins with <b>Logical Wait - And</b>, <b>Logical Wait - Or</b> or <b>Logical Wait - Exclusive Or</b> command, and should end with a <b>Logical Wait End</b> command. An <b>And</b> block will wait until all subqueries are true, an <b>Or</b> block will wait until any subquery is true, and an <b>Exclusive Or</b> block will wait until exactly one subquery is true. Additionally, a logical wait block can be used as a condition for an <a href="#if">if statement</a>. Multiple logical wait blocks can be nested if more complex logic is required. All "Wait for" commands can be used as subqueries, along with the <a href="#role">Set role</a>, <a href="#setplayerweapon">Set player weapon</a>, <a href="#face-towards-like-guard">Face towards like guard</a> and <a href="#set-movement">Set movement type</a> commands, which function as they do when used as an if condition.</li>
+  <li><a name="logical-wait"><b>Logical Waits</b></a> - Logical wait commands can be used to combine multiple queries into a single query. A logical wait block begins with <b>Wait for All</b>, <b>Wait for Any</b> or <b>Wait for Exactly One</b> command, and should end with a <b>Wait for Conditions End</b> command. An <b>All</b> block will wait until all subqueries are true, an <b>Any</b> block will wait until any subquery is true, and an <b>Exactly One</b> block will wait until exactly one subquery is true. Additionally, a logical wait block can be used as a condition for an <a href="#if">if statement</a>. Multiple logical wait blocks can be nested if more complex logic is required. All "Wait for" commands can be used as subqueries, along with the <a href="#role">Set role</a>, <a href="#setplayerweapon">Set player weapon</a>, <a href="#face-towards-like-guard">Face towards like guard</a> and <a href="#set-movement">Set movement type</a> commands, which function as they do when used as an if condition.</li>
```

Comment:
Changes are looking good!  One more suggested change here to the verbiage, shifting the description toward the new naming.

How about changing the first part to something like:

"Multi-expression wait commands are used to perform an atomic check on multiple conditions, i.e., as a single combined boolean query, which returns a single true or false value. Script execution halts at this point until the multiple conditions are met as specified. The expression block begins..." and so forth.